diff --git a/auxiliary_files/CMakeLists.txt b/auxiliary_files/CMakeLists.txt
index d0de137fd..ae204877a 100644
--- a/auxiliary_files/CMakeLists.txt
+++ b/auxiliary_files/CMakeLists.txt
@@ -72,7 +72,7 @@ if(server)
 		)
 		if(NOT SYSUSERS_DIR)
 			# Fallback value, in case it could not be fetched via pkg-config
-			set(SYSUSERS_DIR "${CMAKE_INSTALL_SYSCONFDIR}/sysconfig.d")
+			set(SYSUSERS_DIR "lib/sysusers.d")
 		endif()
 		get_pkgconf_variable(
 			OUTPUT_VARIABLE SYSTEMD_SERVICE_DIR
@@ -81,10 +81,8 @@ if(server)
 			QUIET
 		)
 		if(NOT SYSTEMD_SERVICE_DIR)
-			# Default to /etc/systemd/system - This is not where packages ought to install their service files (rather
-			# it's where an admin is supposed to add their system files), but this seems to be the most consistent
-			# path across different platforms, so we use it anyway.
-			set(SYSTEMD_SERVICE_DIR "${CMAKE_INSTALL_SYSCONFDIR}/systemd/system")
+			# Fallback value, in case it could not be fetched via pkg-config
+			set(SYSTEMD_SERVICE_DIR "lib/systemd/system")
 		endif()
 
 		delayed_configure_files(
