--- mumble-1.2.4/src/mumble/AudioInput.cpp.bero	2012-04-22 22:54:13.627406284 +0200
+++ mumble-1.2.4/src/mumble/AudioInput.cpp	2012-04-22 22:54:37.389406824 +0200
@@ -47,6 +47,10 @@
 #include "opus.h"
 #endif
 
+#ifndef CELT_SET_VBR_RATE
+#define CELT_SET_VBR_RATE CELT_SET_BITRATE
+#endif
+
 // Remember that we cannot use static member classes that are not pointers, as the constructor
 // for AudioInputRegistrar() might be called before they are initialized, as the constructor
 // is called from global initialization.
